### ISSUE TITLE
Switch with number in strings as value -> Error: Arguments must either all be numbers or strings

### ISSUE THREAD

**Issue by cxbuilders** [microsoft/BotFramework-Composer#3361]
When using a switch statement on a string that contains numbers;
![image](https://user-images.githubusercontent.com/49873947/84156438-c4e74400-aa69-11ea-8bac-2add0456fa98.png)

It throws an error:
![image](https://user-images.githubusercontent.com/49873947/84156474-d29cc980-aa69-11ea-8cfc-9027a710044d.png)

When just adding a letter after the 23 like 23A it works ok.

Somewhere the 23 does get converted into a number although I specifically set the Value Type to string.

## Version

858ef680

**Comment by hibrenda** [comment]
if you are using number as below, and then set switch condition as number as well. it should work.
![image](https://user-images.githubusercontent.com/43408889/84256271-444a4580-ab46-11ea-8952-110816d4497b.png)
![image](https://user-images.githubusercontent.com/43408889/84256379-693eb880-ab46-11ea-9986-9fdacd20008e.png)

**Comment by cxbuilders** [comment]
@hibrenda Yeah, but that's not the situation. The condition property is a string that contains a number.  Whether I check on 24 or SomeOtherValue should not matter, as they both should be treated as strings, since that is what the incoming property is and the condition value is.

![image](https://user-images.githubusercontent.com/49873947/84260605-9b3a2580-ab1a-11ea-9e10-95e39337f8d0.png)
![image](https://user-images.githubusercontent.com/49873947/84260627-a3926080-ab1a-11ea-9c0e-f879be6931b0.png)

**Comment by hibrenda** [comment]
@vishwacsena this should be an issue in SDK?
